libsolidity/formal/SMTEncoder.h

```diff
@@ -57,6 +57,7 @@ class SMTEncoder: public ASTConstVisitor
 		ModelCheckerSettings _settings,
 		langutil::UniqueErrorReporter& _errorReporter,
 		langutil::UniqueErrorReporter& _unsupportedErrorReporter,
+		langutil::ErrorReporter& _provedSafeReporter,
```

Comment:
I don't see why we need to add this to SMTChecker.
Should work fine without it, no?

Reply:
Right. That's a mistake.

Reply:
@blishko My previous comment is wrong. They do not keep their own reference. They use protected field from SMTEncoder.